packages/network_info_plus/network_info_plus/example/integration_test/network_info_plus_test.dart

```diff
@@ -19,12 +19,13 @@ void main() {
       _networkInfo = NetworkInfo();
     });
 
-    testWidgets('test location methods, iOS only', (WidgetTester tester) async {
+    testWidgets('test location methods, iOS and MacOS only',
+        (WidgetTester tester) async {
       if (Platform.isIOS) {
```

Comment:
this line should be removed, otherwise it won't run on macos

Reply:
oh my bad. thanks for catching that! didn't expect that to be there 😅

Reply:
turns out, this is only implemented in IOS (as the comment said). reverted. sorry bout that.